presto-docs/src/main/sphinx/release/release-0.246.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.246
+=============
+
+**Highlights**
+==============
+
```

Comment:
Let's include all changes that starts with "Add" and "Improve".

Reply:
I don't think there is anything that major. lets remove this section for this release